mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/ExceptionEventRequest.cs

```diff
@@ -6,7 +6,7 @@ namespace Mono.Debugger.Soft
 	public sealed class ExceptionEventRequest : EventRequest {
 
 		TypeMirror exc_type;
-		bool caught, uncaught, subclasses;
+	    bool caught, uncaught, subclasses, win_features, everything_else;
 		
```

Comment:
Indentation change.

Reply:
This should have some descriptive name instead of win_features.

Reply:
Done